media/css/quantum/_buttons.scss

```diff
@@ -2,10 +2,35 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-@import '../../../pebbles/includes/lib';
+@import '../pebbles/includes/lib';
+@import 'lib';
 
-$color-quantum-bright-green: #4ee331;
-$color-quantum-blue: #0a84ff;
+
+// update style of nav/sub nav download buttons
+#nav-download-firefox,
+#global-nav-download-firefox,
+#sub-nav-download-firefox {
+    .download-link.button.button-green {
```

Comment:
I'm guessing we're not going to change the default button styles for the nav on all pages yet, but It would be nice to make this a new button style, instead of just overriding `button-green`?

Reply:
For now, assuming any template that includes this file will want the full "quantum" UI experience. Can/will revisit in upcoming PRs if this proves to not be the case.